docs/_documentations/remotedeploy-single.md

```diff
@@ -72,13 +72,13 @@ A complete command might look something like this :
 --ingress mycluster-12345-7674b4bd9abbdeea0be228236d5275c9-0001.eu-gb.containers.appdomain.cloud
 ```
 
-Which will:
+Which:
```

Comment:
If we translate this file into another language in the future, I'm thinking "which" might be confusing. A complete sentence like "This command performs the following actions:" might be clearer...